OpenSim/Simulation/Model/TwoFrameLinker.h

```diff
@@ -599,13 +598,16 @@ void TwoFrameLinker<C, F>::updateFromXMLNode(SimTK::Xml::Element& aNode,
                 XMLDocument::addPhysicalOffsetFrame30505(aNode,
                         frame1_connectee_name,
                         frame1Name, locationInFrame1, orientationInFrame1);
+            }
+            // Prior to 4.0, all bodies lived in the Model's BodySet and
```

Comment:
If this is true for all pre 4.0 models I'd change the test to remove reference to version 30505 which is not supported and refer to 30516 instead

Reply:
I'm OK with the 30505 reference since this is when the change was made. This is not user-facing code so there's no reason for them to know about it. Changing it now is a bit like rewriting history and it has no impact on users or model files generated in 4.0.

Reply:
@aseth1 by modifying this block you're rewriting history already. The typical use of these version upgrades is that you don't modify the old function (as done here) but instead write a new one to perform the incremental change to previous to current version that's why I'm suggesting you use 30516 which we agreed to maintain so future developers looking into this code don't miss it as an unsupported version.

Reply:
I don't agree with the statement that the modifications here are changing history, but since it doesn't matter anyways I will make the change to get this PR through.

Reply:
As compromise suggest by @chrisdembia I'll rename `XMLDocument::addPhysicalOffsetFrame30505` to  `XMLDocument::addPhysicalOffsetFrame30505_30517()` since 30505 introduced the `PhysicalOffsetFrame` to embody the transform and 30517 to to reflect the update to the pathname.